app/presenters/publishing_api_presenters/edition.rb

```diff
@@ -37,10 +37,6 @@ def as_json
 
   private
 
-    # An incomplete and temporary details hash that only currently includes
```

Comment:
It's out of date, and that information is pretty well conveyed by the other two paragraphs and all the commit messages.